code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -31,6 +30,78 @@
 		else
 			update_inv_r_hand(FALSE)
 
+/mob/living/carbon/human/proc/do_kb_equip_slot_s_store()
+	do_kb_equip(SLOT_S_STORE,COMSIG_KB_SLOT_S_STORE)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_wear_suit()
+	do_kb_equip(SLOT_WEAR_SUIT,COMSIG_KB_SLOT_WEAR_SUIT)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_belt()
+	do_kb_equip(SLOT_BELT,COMSIG_KB_SLOT_BELT)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_back()
+	do_kb_equip(SLOT_BACK,COMSIG_KB_SLOT_BACK)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_boot()
+	do_kb_equip(SLOT_IN_BOOT,COMSIG_KB_SLOT_BOOT)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_head()
+	do_kb_equip(SLOT_IN_HEAD,COMSIG_KB_SLOT_HEAD)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_l_store()
+	do_kb_equip(SLOT_L_STORE,COMSIG_KB_SLOT_L_STORE)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_r_store()
+	do_kb_equip(SLOT_R_STORE,COMSIG_KB_SLOT_R_STORE)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_accessory()
+	do_kb_equip(SLOT_IN_ACCESSORY,COMSIG_KB_SLOT_ACCESSORY)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_beltholster()
+	do_kb_equip(SLOT_IN_HOLSTER,COMSIG_KB_SLOT_BELT_HOLSTER)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_suitsholster()
+	do_kb_equip(SLOT_IN_S_HOLSTER,COMSIG_KB_SLOT_SLOT_S_HOLSTER)
+
+/mob/living/carbon/human/proc/do_kb_equip_slot_backholster()
+	do_kb_equip(SLOT_IN_B_HOLSTER,COMSIG_KB_SLOT_B_HOLSTER)
+
+/mob/living/carbon/human/proc/do_kb_equip(KB,COMSIG)
+	. = COMSIG //The return value must be a flag compatible with the signals triggering this.
```

Comment:
COMSIG what

Reply:
its for the proc right above it. it changes based on what key calls it.

Reply:
Make args lowercase

Reply:
you arent even returning any flags